scripts/measure_for_automatic.py

```diff
@@ -0,0 +1,99 @@
+import os
+
+import json
+import h5py
+import numpy as np
+import pandas as pd
+import imageio.v3 as imageio
+
+from traccuracy import run_metrics
+from traccuracy.matchers import CTCMatcher
+from traccuracy._tracking_graph import TrackingGraph
+from traccuracy.metrics import CTCMetrics, DivisionMetrics
+from traccuracy.loaders._ctc import _get_node_attributes, ctc_to_graph, _check_ctc
+
+
+ROOT = "/scratch/share/cidas/cca/experiments/results_for_automatic_tracking"
+
+
+def _get_tracks_to_isbi():
+    with open(os.path.join(ROOT, "tracking_result.json"), "r") as f:
+        data = json.load(f)
+
+    segmentation = imageio.imread(os.path.join(ROOT, "tracking_result.tif"))
+
+    # we convert the parent ids to frames
+    track_info = {}
+    for parent_id, daughter_ids in data.items():
```

Comment:
You only add the information for tracks that are part of a lineage that has at least one division.
Lineages without a division are not stored in the `tracking_result.json`. Hence there is a lot of info missing here.

Reply:
It should be enough to do a `np.unique` over the tracks that are in `segmentation` and compare with the ids that are in the `tracking_result.json`, and write out extra tracks.

Reply:
Okay, got it. I'll create a new PR for this which takes care of this issue.